server/src/test/java/org/cloudfoundry/identity/uaa/zone/GeneralIdentityZoneConfigurationValidatorTests.java

```diff
@@ -358,4 +360,18 @@ public void validate_invalid_corsPolicy_defaultConfiguration_allowedOrigins() th
         expection.expectMessage("Invalid value in config.corsPolicy.defaultConfiguration.allowedOrigins: '^/uaa/userinfo('");
         validator.validate(zone, mode);
     }
+
+    @Test
+    public void validate_with_token_key_active() throws InvalidIdentityZoneConfigurationException {
+        Map<String, TokenPolicy.KeyInformation> keyInformationMap = new HashMap<>();
+        TokenPolicy.KeyInformation keyInformation = new TokenPolicy.KeyInformation();
+        keyInformation.setSigningKey(legacyKey);
+        keyInformation.setSigningCert(legacyCertificate);
+        keyInformation.setSigningAlg("RS256");
+        keyInformationMap.put("id-1", keyInformation);
+        zone.getConfig().getTokenPolicy().setKeyInformation(keyInformationMap);
+        zone.getConfig().getTokenPolicy().setActiveKeyId("id-1");
+
+        validator.validate(zone, mode);
```

Comment:
I'm guessing that the test is testing that this call doesn't throw an exception. 

If that's the case, how would you feel about using [assertDoesNotThrow](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertDoesNotThrow(org.junit.jupiter.api.function.Executable)), I think that it makes the test more readable.

Reply:
This part is outdated